app/script/components/messageQuote.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+'use strict';
+
+window.z = window.z || {};
+window.z.components = z.components || {};
+
+z.components.MessageQuote = class MessageQuote {
+  constructor({quote, conversationRepository, locationRepository}) {
+    this.updateCanShowMore = this.updateCanShowMore.bind(this);
+    this.toggleShowMore = this.toggleShowMore.bind(this);
+
+    const activeConversation = conversationRepository.active_conversation;
```

Comment:
Since we don't need the power of the `conversationRepository` I would argue that we could just inject a `conversationEntity` as a param of the component.

The less the component knows about the app architecture the more future proof we will be

Reply:
Here is a suggestion of signature that could work if I am not mistaken

```
<message-quote param="message: message, locationRepo: locationRepo"></message-quote>
```